openqabot/types/incidents.py

```diff
@@ -135,10 +137,11 @@ def __call__(
                             ArchVer(arch, self.settings["VERSION"])
                         ]
                     except KeyError:
-                        logger.debug(
-                            "Incident %s does not have %s arch in %s"
-                            % (inc.id, arch, self.settings["VERSION"])
-                        )
+                        if REALLY_LONG_LOG:
```

Comment:
What is this about?

Reply:
trace, usually this info has no value , only when is needed detailed info

Reply:
That's how I would describe *debug* 🤔

Reply:
I suggest to either just use debug or introduce a trace level like in https://www.programcreek.com/python/example/56843/logging.TRACE